docs/source/getting-started.rst

```diff
@@ -100,36 +100,6 @@ Run your project
 
 .. _Panel: https://panel.holoviz.org
 
-Sharing your project
-====================
-
-To share this project with a colleague:
-
-#. Archive the project::
-
-     $ anaconda-project archive interactive.zip
-
-#. Send the archive file to your colleague.
-
-You can also share a project by uploading it to Anaconda Cloud.
-For more information, see :doc:`user-guide/tasks/share-project`.
```

Comment:
Actually, this is the only portion that needs to be removed. Creating a zip and sharing by means other than anaconda.org should still be documented.

Reply:
Good point, will fix 👍